package.json

```diff
@@ -46,7 +46,7 @@
   },
   "dependencies": {
     "@babel/runtime": "^7.0.0",
-    "mapbox-gl": "~0.53.0",
+    "mapbox-gl": "~1.0.0",
```

Comment:
We will go with a new major release. Let's relax it to `^1.0.0`.

Reply:
Updated!